src/app/zap-templates/templates/app/helper.js

```diff
@@ -196,8 +196,8 @@ function asChipUnderlyingType(label, type)
 
 // List of all cluster with generated functions
 var endpointClusterWithInit =
-    [ 'Basic', 'Identify', 'Groups', 'Scenes', 'Occupancy Sensing', 'On/off', 'Level Control', 'Color Control', 'IAS Zone' ];
-var endpointClusterWithAttributeChanged = [ 'Identify', 'Door Lock' ];
+    [ 'Basic', 'Identify', 'Groups', 'Scenes', 'Occupancy Sensing', 'On/off', 'Level Control', 'Color Control', 'IAS Zone', 'Pump Configuration and Control' ];
+var endpointClusterWithAttributeChanged = [ 'Identify', 'Door Lock', 'Pump Configuration and Control' ];
```

Comment:
Isn't `emberAfPostAttributeChangeCallback` enough? `emberAfPumpConfigurationAndControlClusterServerAttributeChangedCallback` is currently only used for debug logs. Are you planning to add the actual logic there?

Reply:
Yes, we expect to do some cluster specific logic. But perhaps we might not need it in the future.

Reply:
But is it the correct way of implementing general behavior for the PCC? I have a bit of difficulty to figure out where to put generic PCC handling code?

Reply:
There are issues building the all-clusters-app and the QEMU. It does not seem to include the PCC cluster file (.cpp) in the Datamodel. It seems a bit confusing.

Reply:
This is the right way of adding this if we want custom cluster logic on attribute changes.  I didn't obviously see anything in the cluster spec that would need that, but maybe I just missed it?

> It does not seem to include the PCC cluster file (.cpp) in the Datamodel. 

Indeed.  That file did not exist when PCC was added to the .zap file for all-clusters-app, and when the cluster cpp was added no one added it to the all-clusters-app build config.  Presumably it should be added to at least `examples/all-clusters-app/esp32/main/CMakeLists.txt`....  Not sure about the Linux version and how it decides what to include.